src/main/webapp/jsp/instructorCourseEnrollResult.jsp

```diff
@@ -19,8 +19,8 @@
 </c:set>
 
 <ti:instructorPage pageTitle="TEAMMATES - Instructor" bodyTitle="Enrollment Results for ${data.courseId}" jsIncludes="${jsIncludes}">
-    <div class="alert alert-success">
-        <form name='goBack' action="${data.instructorCourseEnrollLink}" method="post" role="form"> 
```

Comment:
`<form name='goBack' action="${data.instructorCourseEnrollLink}" method="post" role="form" > 
            <div class="alert alert-success">
				Enrollment Successful. Summary given below. Click <a id="edit_enroll" href="javascript:document.forms['goBack'].submit()">here</a> to do further changes to the student list.
				<input type="hidden" name="${COURSE_ID}" value="${data.courseId}">
				<input type="hidden" name="${STUDENTS_ENROLLMENT_INFO}" value="${fn:escapeXml(data.enrollStudents)}">
			</div>
		</form>`
this is working..